### PR TITLE
Rum

### PR DESCRIPTION
**Pull request by guperrot**
No iOS support for now.

### CODE REVIEW

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.Droid/project.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
-{
```
Comment (dhei) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.Droid/project.json]: I wonder if this was a file encoding change? Such as UTF-8 to UTF-8 BOM.

File: SDK/MobileCenterRum/Microsoft.Azure.Mobile.Rum.Windows.Shared/RealUserMeasurements.cs [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+            }
+        }
+
+        private void DisposeCancellationSource()
+        {
+            _cancellationTokenSource?.Dispose();
+            _cancellationTokenSource = null;
+        }
+
+        private async Task RunTestsAsync(string rumKey, CancellationToken cancellationToken)
```
Comment (dhei) [inline SDK/MobileCenterRum/Microsoft.Azure.Mobile.Rum.Windows.Shared/RealUserMeasurements.cs]: General feedback - this method is too long, we need to refactor it in the future.

Comment (guperrot) [reply SDK/MobileCenterRum/Microsoft.Azure.Mobile.Rum.Windows.Shared/RealUserMeasurements.cs]: Done in #ab6c7b1

File: SDK/MobileCenterRum/Microsoft.Azure.Mobile.Rum.Windows.Shared/RealUserMeasurements.cs [hunk 3]
```diff
@@ -199,180 +203,209 @@ private void DisposeCancellationSource()
...
+        private async Task<string> GetRemoteConfigurationAsync(CancellationToken cancellationToken, IHttpNetworkAdapter httpNetworkAdapter)
+        {
+            foreach (var url in ConfigurationEndpoints)
+            {
+                try
+                {
+                    MobileCenterLog.Verbose(LogTag, "Calling " + url);
+                    return await httpNetworkAdapter.SendAsync(url, HttpMethod.Get, Headers, "", cancellationToken);
+                }
+                catch (OperationCanceledException)
```
Comment (dhei) [inline SDK/MobileCenterRum/Microsoft.Azure.Mobile.Rum.Windows.Shared/RealUserMeasurements.cs]: Might be a good case to use C# 6 exception filters?

`try {...} catch (Exception e) when (!(e is OperationCanceledException)) {...}`

File: SDK/MobileCenterRum/Microsoft.Azure.Mobile.Rum.Windows.Shared/RealUserMeasurements.cs (hunk 2 above)
Comment (dhei) [reply SDK/MobileCenterRum/Microsoft.Azure.Mobile.Rum.Windows.Shared/RealUserMeasurements.cs]: Looks fine to me.

File: SDK/MobileCenterRum/Microsoft.Azure.Mobile.Rum.Windows.Shared/RealUserMeasurements.cs (hunk 3 above)
Comment (guperrot) [reply SDK/MobileCenterRum/Microsoft.Azure.Mobile.Rum.Windows.Shared/RealUserMeasurements.cs]: Fixed

**Pull request merged**